darts/tests/test_timeseries.py

```diff
@@ -810,6 +810,50 @@ def test_append_values(self):
         assert appended.time_index.equals(expected_idx)
         assert appended.components.equals(series.components)
 
+    def test_pre_and_append_values_empty(self):
+        series = linear_timeseries(start=1, length=9, freq=2)
+        appended = series.append_values([])
+        assert np.allclose(appended.values(), series.values())
+        appended = series.prepend_values([])
+        assert np.allclose(appended.values(), series.values())
+
+    def test_pre_and_append_values_shape(self):
```

Comment:
after our discussion, we probably have to adapt here a bit. 
We only naively expand an input array-like to 3-D with the `expand_arr()` function, assuming that the input dimension follow the shape (time, optional(component), optional(samples)).